### PR TITLE
Fixing Download behavior in tests with docker

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.6.x
| Description?  | Fixing Download behavior in tests with docker
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | Run tests with docker

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/13900)

### CODE REVIEW

**Comment by boubkerbribri** [comment]
@PierreRambaud , Can you check please why continuous-integration/travis-ci/pr is failed ?

File: tests/E2E/test/common.webdriverio.js [hunk 1]
```diff
@@ -180,19 +192,19 @@ module.exports = {
       options.host = global.selenium_host;
     }
     else {
-      // We need to set it to localhost for "got" library
+      // We need to set it to localhost for 'chai-http' library
```
Comment (PierreRambaud) [inline tests/E2E/test/common.webdriverio.js:193]: You're not using chai-http :thinking:

**Comment by PierreRambaud** [comment]
@boubkerbribri Done :wink:

File: tests/E2E/test/common.webdriverio.js (hunk 1 above)
Comment (boubkerbribri) [reply tests/E2E/test/common.webdriverio.js:193]: I replace got by chai-http in PR #13811 , like you suggested

Comment (PierreRambaud) [reply tests/E2E/test/common.webdriverio.js:193]: Cooooool :+1:

**Review by PierreRambaud — APPROVED** [review]

**Comment by mbadrani** [comment]
Thanks @boubkerbribri

**Pull request merged**